CRM/Activity/Tokens.php

```diff
@@ -36,6 +36,19 @@ class CRM_Activity_Tokens extends CRM_Core_EntityTokens {
 
   use CRM_Core_TokenTrait;
 
+  /**
+   * Class constructor.
+   *
+   * Overriding because the trait needs this to happen & trying to
+   * leave any changes that affect the trait out of scope here.
+   */
+  public function __construct() {
```

Comment:
I just put this here because otherwise discussing what to do about the trait & the weird support for that multi-syntax needs to be a separate discussion